modular_skyrat/modules/food_replicator/code/clothing.dm

```diff
@@ -0,0 +1,59 @@
+/obj/item/clothing/under/colonial
+	name = "colonial outfit"
+	desc = "Fancy white satin shirt and a pair of cotton-blend pants with a black synthleather belt."
+	icon = 'modular_skyrat/modules/food_replicator/icons/clothing.dmi'
+	worn_icon = 'modular_skyrat/modules/food_replicator/icons/clothing_worn.dmi'
+	worn_icon_digi = 'modular_skyrat/modules/food_replicator/icons/clothing_digi.dmi'
+	icon_state = "under_colonial"
+
+/obj/item/clothing/under/colonial/mob_can_equip(mob/living/M, slot, disable_warning, bypass_equip_delay_self, ignore_equipped, indirect_action)
+	if(is_species(M, /datum/species/teshari))
+		to_chat(M, span_warning("[src] is far too big for you!"))
+		return FALSE
+	return ..()
```

Comment:
I'm not going to do it for all of the ones in this file, but please, single-letter variables are bad, and leave an empty line when you go down one (or more) levels of indentation.
```suggestion
/obj/item/clothing/under/colonial/mob_can_equip(mob/living/equipper, slot, disable_warning, bypass_equip_delay_self, ignore_equipped, indirect_action)
	if(is_species(equipper, /datum/species/teshari))
		to_chat(equipper, span_warning("[src] is far too big for you!"))
		return FALSE

	return ..()
```

Reply:
Yeah I've copied some of the old code; gonna do it tomorrow at this point.